js/models.js

```diff
@@ -0,0 +1,32 @@
+App.Models.Todo = Backbone.Model.extend({
+    defaults: {
+        'isComplete': false,
```

Comment:
Also, generally, don't wrap quotes around object properties unless it's required (they use `-` etc). Most of our linters will warn on this, but in general it's good to know.